src/util/safelywritablefile.cpp

```diff
@@ -46,40 +56,10 @@ SafelyWritableFile::SafelyWritableFile(QString origFileName, bool useTemporaryFi
         // Abort constructor
         return;
     }
-    if (useTemporaryFile) {
-        QString tempFileName = origFileName + kSafelyWritableTempFileSuffix;
-        QFile origFile(origFileName);
-        if (!origFile.copy(tempFileName)) {
```

Comment:
This copy is lost now.

Reply:
I have made an method exactly the way it is.
That should be the copy.
https://github.com/mixxxdj/mixxx/blob/9a52cd354ef840ffd924a3a87358447fea1b20cf/src/util/safelywritablefile.cpp#L281